index.html

```diff
@@ -43,7 +43,7 @@
         <a id="header-menu" class="icon-menu"></a>
 
         <ul  id="main-nav" class="the-nav">
```

Comment:
Adding some hover effects here would help with the overall theme of the site.  Similar to the social media links at the bottom of the page.